riscv/insns/sd.h

```diff
@@ -1,2 +1,9 @@
-require_rv64;
-MMU.store<uint64_t>(RS1 + insn.s_imm(), RS2);
+if (xlen == 32)
+{ // sd - Store pair RV32
+    require_extension(EXT_ZILSD);
+    require(insn.rs2() % 2 == 0);
+} else {
+    require(xlen == 64);
+}
+
+STORE_D;
```

Comment:
Please delete the `STORE_D` macro and move that code back into this position.

Reply:
(Or, rather, move its two pieces into the respective branches of the preceding if/else statement.)

Reply:
done in 02fbb0d